app/assets/javascripts/import_records/ImportRecordToggleableSection.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export default class ImportRecordToggleableSection extends React.Component {
+
+  render() {
+    const {title} = this.props;
+    const titleStyle = {
+      marginTop: 20,
+      marginBottom: 10,
+      borderBottom: '1px solid #ccc'
+    };
+
+    return (
+      <div>
+        <div style={titleStyle}>{title} {this.renderToggle()}</div>
+        {this.renderContent()}
+      </div>
+    );
+  }
+
+  renderContent() {
+    const {shouldShow, hasNoData, children} = this.props;
+
+    if (hasNoData === true) return null;
+    if (shouldShow === false) return null;
+
+    const preStyle = {
+      fontSize: 14,
+      color: '#3d3d3d',
+      marginTop: 10,
+      marginBottom: 20,
+    };
+
+    return (
+      <pre style={preStyle}>{children}</pre>
```

Comment:
If I read right, the point of this component is to render preformatted text and hide/show it.  So this doesn't seem like `children` is the right API - would it ever make sense to put other React elements inside here other than plain text?

Reply:
Ooooh good catch!